pkgs/tools/text/sift/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "sift-${version}";
+  version = "0.8.0";
+  rev = "v${version}";
+
+  goPackagePath = "github.com/svent/sift";
+
+  src = fetchFromGitHub {
+    inherit rev;
+    owner = "svent";
+    repo = "sift";
+    sha256 = "1nb042k420xr6000ipwhqn41vg8jfp6ghq4z7y1sjnndkrhclzm1";
+  };
+
+  goDeps = ./deps.json;
+}
```

Comment:
Could you please add a meta section here? This is also where you would set yourself as maintainer of that package, as you're adding yourself to the maintainers list. ;)